release.properties

```diff
@@ -0,0 +1,16 @@
+#release configuration

```

Comment:
This file should not be part of the source repository.  please update your .gitignore appropriately